dwh/es_data_marts/README.md

```diff
@@ -1,15 +1,13 @@
-Welcome to your new dbt project!
+# dbt project for data marts
```

Comment:
I would suggest to rename the directory from `es_data_marts` to `dbt`, or `bigquery_dbt`, or something else to that effect, because the current name implies that the data is loaded into Elastic directly (but in reality it all happens within BigQuery, and only then loaded into Elastic)